.Net Core Agent provider

ChNorman opened the pull request:
updated with lwrp to install New Relic .Net Core Agent

ChNorman commented:
Resolves Issue #359

djoos commented:
Hi @ChNorman,

thanks for this PR - is there any chance to rebase the PR so it only includes the New Relic .Net Core Agent changes? It seems other changes are included as well currently...

Thanks in advance!
David

ChNorman commented:
hi @djoos 

I could attempt to but outside of the changes for the .net core the only other thing in the pull request was changes required to fet integrated testing to pass and to get the request to pass your CI build

ChNorman commented:
rebased as requested but it looks like some current robocop rules are casuing CI  integration to fail. Please let me know how you would like to address because some of the failures are out side code that I modified
